### DISCUSSION TITLE
Is there a way of using subscriptions on iOS 11-12?

### DISCUSSION THREAD

**Discussion by rndhacker** [maticzav/swift-graphql#75] [Q&A]
Hello, I have been using your library and it works very well! I wanted to ask if you guys have found a way to make subscriptions in lower Operating systems (11 and 12 to be precise) since the Apple WS implementation is using iOS 13+ and my app needs to support lower OS. I would be happy to contribute if you have thoughts or any ideas.

**Comment by maticzav** [comment]
Hi!

We haven't checked if we can further lower the OS requirement, but I don't think it should be impossible. Would love to see you stab this and happily help you merge the improvement. 🙂
